content/en/serverless/agent_configuration/_index.md

```diff
@@ -0,0 +1,138 @@
+| `DD_LOGS_CONFIG_FILE_WILDCARD_SELECTION_MODE` | The strategy used to prioritize wildcard matches if they exceed the open file limit. Choices are `by_name` and `by_modification_time`.                                                                                               |
+
+### APM configuration
+
+| Env Variable              | Description                                                                                                                                                                                                                           |
+|---------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `DD_APM_ENABLED`          | Set to true to enable the APM Agent. Default true.                                                                                                                                                                                    |
+| `DD_APM_ENV`              | The environment tag that Traces should be tagged with. If not set the value will be inherited, in order, from the top level "env" config option if set and then from the 'env:' tag if present in the 'tags' top level config option. |
+| `DD_APM_RECEIVER_PORT`    | The port that the trace receiver should listen on. Set to 0 to disable the HTTP receiver. Default: 8126                                                                                                                               |
+| `DD_APM_RECEIVER_SOCKET`  | Accept traces through Unix Domain Sockets. It is off by default. When set, it must point to a valid socket file.                                                                                                                      |
+| `DD_APM_DD_URL`           | Define the endpoint and port to hit when using a proxy for APM. The traces are forwarded in TCP therefore the proxy must be able to handle TCP connections. <ENDPOINT>:<PORT>                                                         |
+| `DD_APM_REPLACE_TAGS`     | Defines a set of rules to replace or remove certain resources, tags containing potentially sensitive [information][3].                                                                                                                |
+| `DD_APM_IGNORE_RESOURCES` | An exclusion list of regular expressions can be provided to disable certain traces based on their resource name all entries must be surrounded by double quotes and separated by commas.                                              |
+| `DD_APM_LOG_FILE`         | The full path to the file where APM-agent logs are written.                                                                                                                                                                           |
```

Comment:
Not sure if the APM log file location will be particularly relevant, since customers won't have a way to easily collect it.